SMARTAGRIFOOD/vineyardData/examples/example-normalized.json

```diff
@@ -1,36 +1,125 @@
-{
-	"id": "VB638235",
-	"type": "AgriVineyardBlock",
-	"name": {
-		"value": "Northern block",
-		"type": "String"
-	},
-	"dateModified": {
-		"value": "2021-09-29T00:00:00.0Z",
-		"type": "DateTime"
-	},
-	"inclination": {
-		"value": {
-			"aspect": {
-				"value": "E",
-				"type": "String"
-			},
-			"slope": {
-				"value": "2",
-				"type": "String"
-			}
-		}
-	},
-	"rowSpacing": {
-		"value": 2,
-		"type": "Float"
-	},
-	"plantingStyle": {
-		"value": "rows",
-		"type": "String"
-	},
-	"dateCreated": {
-		"value": "2015-02-28T00:00:00.0Z",
-		"type": "DateTime"
-	}
-}
+{
+	"id": "VB638235",
+	"type": "AgriVineyardBlock",
+	"name": {
+		"value": "Northern block",
+		"type": "String"
+	},
+	"coordinates": {
+		"value": "[123,234,345]",
+		"type": "String"
+	},
+	"address": {
+		"value": "Orehovci 1a, 9250 Gornja Radgona, Slovenia",
+		"type": "String"
+	},
+	"size": {
+		"value": 3014,
+		"type": "Float"
+	},
+	"numVinesRepresented": {
+		"value": 237,
+		"type": "Integer"
+	},
+	"plants": {
+		"value": {
+			"trainingSystem": {
+				"value": "Double cordon",
+				"type": "String"
+			},
+			"variety": {
+				"value": "Chardonnay",
+				"type": "String"
+			},
+			"clone": {
+				"value": "76",
+				"type": "String"
+			},
+			"rootstock": {
+				"value": "3309 Couderc",
+				"type": "String"
+			}
+		}
+	},
+	"microclimate": {
+		"value": {
+			"airTemperatureTSA": {
+				"value": {
+					"minOverTime": {
+						"value": 10.2,
+						"type": "Float"
+					},
+					"maxOverTime": {
+						"value": 25.3,
+						"type": "Float"
+					},
+					"avgOverTime": {
+						"value": 19.4,
+						"type": "Float"
+					}
+
+				}
+			},
+			"dailyGDD": {
+				"value": 25.3,
+				"type": "Float"
+			},
+			"dailyRainfall": {
+				"value": 120,
+				"type": "Float"
+			},
+		}
+	},
+
+	"hasAgriSoil":{

```

Comment:
misleading attribute name. Looks like a relationship but it is a string value... there are better options. See [guidelines](https://github.com/smart-data-models/data-models/blob/master/guidelines.md)

Reply:
this was copied from https://github.com/smart-data-models/dataModel.Agrifood/blob/master/AgriParcel/schema.json with the goal to be identical.
should we just keep 2nd available type (uri)?